contracts/core/State.sol

```diff
@@ -228,4 +230,9 @@ contract State is Objects, ReentrancyGuard, SharedReentrancyGuard, Ownable {
             logicTargetsSet.removeBytes32(bytes32(sig));
         }
     }
+
+    modifier onlyPauserOrOwner() {
```

Comment:
pls move the modifier to where it is actually used for clarity - ProtocolSettings

Reply:
i'd prefer to keep it in the state contract, so that it can be reusable by other module

Reply:
ok. makes sense.